airbyte-webapp/src/locales/en.json

```
@@ -196,11 +196,11 @@
   "onboarding.exploreDemo": "Explore our demo app with test data",
   "onboarding.firstConnection": "Set up your first connection",
   "onboarding.createFirstSource": "Create your <name>first source</name>",
-  "onboarding.createFirstSource.text": "Sources are tools where the data will be replicated from. ",
+  "onboarding.createFirstSource.text": "Sources are connectors that pull data from a data source.",
   "onboarding.createFirstDestination": "Create your <name>first destination</name>",
-  "onboarding.createFirstDestination.text": "Sources are tools where the data will be replicated from. ",
+  "onboarding.createFirstDestination.text": "Destinations are connectors that pushes data to a data source.",
```

```suggestion
  "onboarding.createFirstDestination.text": "Destinations are connectors that push data to a data store.",
```
sorry this would be better than the original copy I gave you